### ISSUE TITLE
Add keybinding for adding a category

### ISSUE THREAD

**Issue by taniarascia** [taniarascia/takenote#11]
Figure out why global keybindings can't access local React hook state.

**Comment by acagastya** [comment]
Why not add accesskey to the button?  Is there a reason why not to do it?

How about this in `AppSideBar.tsx`?
```diff
-<button className="add-category-button" onClick={newTempCategoryHandler}>
+<button className="add-category-button" onClick={newTempCategoryHandler} accessKey='c'>
```

**Comment by taniarascia** [comment]
Fixed in #21
